src/main/java/projet/projetstage02/exception/NotReadyToBeSigneException.java

```diff
@@ -0,0 +1,5 @@
+package projet.projetstage02.exception;
+
+public class NotReadyToBeSigneException extends Exception {
+    public NotReadyToBeSigneException(){super("The contract isn't ready to be signe be the gestionnaire!");}
```

Comment:
Ligne 3-4: Petite erreur, devrait etre "signed" pas "signe".

Reply:
"Signed" is in the past, "Signe" is in the present I want to signe the contract now but I can't because there are other signature missing